src/prpy/planning/cbirrt.py

```diff
@@ -163,7 +164,7 @@ def PlanToTSR(self, robot, tsr_chains, smoothingitrs=100, **kw_args):
 
     def Plan(self, robot, smoothingitrs=None, timelimit=None, allowlimadj=0,
              jointstarts=None, jointgoals=None, psample=None, tsr_chains=None,
-             extra_args=None, **kw_args):
+             extra_args=None, save_mimic_trajectories=False, **kw_args):
         """
         @param allowlimadj If True, adjust the joint limits to include
```

Comment:
Couldn't this also take a dict as an argument and just save the trajectory in there, i.e. a user-supplied out-arg? Then the user could do what they want with the trajectory.